src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import duke.command.Command;
+import duke.common.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Runs a Personal Assistant Chat bot.
+ */
 public class Duke {
+
+    /** Handles loading and saving of tasks. */
+    private Storage storage;
+
+    /** Temporary store of list of tasks. */
+    private TaskList tasks;
```

Comment:
Great use of the plural noun `tasks` to show that `TaskList` encapsulates multiple `Task` objects.

Reply:
Thank you!